lang/en/texts/open-food-facts-mobile-app.html

```diff
@@ -1,113 +1,213 @@
-New features: continuous scan, offline mode, product comparison, lists, unlimited history and more !
-</p>
-<p><b>2012:</b> <a href="https://blog.openfoodfacts.org/fr/news/application-mobile-open-food-facts-pour-android" alt="Blogpost about the pioneering Open Food Facts mobile app">Launch of the Open Food Facts mobile app, the very first collaborative scanning app for food.</a>
-</p>
-</div>
-<!-- TODO: Translate images to your language. Ask @teolemon for the SVG file-->
-<div class="row">
-<div class="small-12 medium-12 large-6 columns">
-<img src="https://static.openfoodfacts.org/images/misc/mobileapp/new_openfoodfacts_app_en_light_3_screenshots.png" alt="Try the new Open Food Facts mobile app">
-</div>
-</div>
-
-<!-- DO NOT translate after this line -->
-
-<scripts>
-<script>
-function getMobileOperatingSystem() {
-  var userAgent = navigator.userAgent || navigator.vendor || window.opera;
-
-      // Windows Phone must come first because its UA also contains "Android"
-    if (/windows phone/i.test(userAgent)) {
-        return "Windows Phone";
-    }
-
-    if (/android/i.test(userAgent)) {
-        return "Android";
-    }
-
-    // iOS detection from: https://stackoverflow.com/a/9039885/177710
-    if (/iPad|iPhone|iPod/.test(userAgent) && !window.MSStream) {
-        return "iOS";
-    }
-
-    return "unknown";
-}</script>
-
-<!-- You can change the URLs to one that works in your language. If you are unsure, do not touch anything -->
-
-<script>
-function DetectAndServe(){
-
-if (getMobileOperatingSystem() == "Android") {
-    window.location.href = "https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=en";
-    }
-if (getMobileOperatingSystem() == "iOS") {
-    window.location.href = "https://apps.apple.com/app/open-food-facts/id588797948";
-    }
-if (getMobileOperatingSystem() == "Windows Phone") {
-    window.location.href = "https://apps.microsoft.com/store/detail/open-food-facts-scan-to-get-nutriscore-ecoscore-and-more/XP8LT18SRPKLRG";
-    }
-};
-DetectAndServe();
-</script>
-</scripts>
-
-<initjs>
-</initjs>
+<div id="app-landing-page">
+    <div id="download-app-block" class="block block-blue">
+        <div class="row">
+            <div class="small-12 medium-4 medium-offset-1 column">
+                <h1 class="title-2 emphasized-title">
+                    Choose products that are good for
+                    <span class="highlight-label yellow rotate-left">you</span>
+                    and the
+                    <span class="highlight-label blue rotate-right">planet</span>
+                </h1>
+                <p>The world's largest open food database at your fingertips!</p>
+                <a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=en">
```

Comment:
```suggestion
                <a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=en" title="Open Food Facts on Google Play">
```

Reply:
is it something usually advised for accessibility?

Reply:
no idea: https://www.w3.org/WAI/WCAG22/Techniques/html/H33.html